### ISSUE TITLE
Reflection outputs broken JavaScript metadata for methods with ref arguments

### ISSUE THREAD

**Issue by TriEdgeAI** [bridgedotnet/Bridge#2071]
**C# code:**
```csharp
class RoutePatternParser
{
    public Regex Parse(string pattern, out List<string> tokens)
    {
        //Removed for brevity.
    }
}
```

**Output metadata:**
```js
    Bridge.setMetadata(Sirius.Web.RoutePatternParser, function() {
        return {
            "members": [{
                "accessibility": 2,
                "name": "Parse",
                "type": 8,
                "paramsInfo": [{
                    "name": "pattern",
                    "parameterType": String,
                    "position": 0
                }, {
                    "name": "tokens",
                    "isOut": true,
                    "parameterType": System.Collections.Generic.List & ,
                    "position": 1
                }],
                "sname": "Parse",
                "returnType": System.Text.RegularExpressions.Regex,
                "params": [String, System.Collections.Generic.List & ]
            }, }]
        };
    });
```

Notice this part:
```js
"parameterType": System.Collections.Generic.List & ,
```

**Comment by vladsch** [comment]
It is already fixed in 15.4

**Comment by TriEdgeAI** [comment]
Oh, cool. Sorry.
